packages/react/src/components/form/initial.tsx

```diff
@@ -157,18 +167,42 @@ const HandleForm: React.FC<HandleFormProps> = ({
         label={text["initial.handle.email"]}
         placeholder={text["initial.handle.phone.email"]}
         value={values[handleType] ?? ""}
-        onChange={registerField(handleType, (value) => {
-          if (!isValidEmail(value)) {
-            return { message: text["validationError.email"] };
-          }
+        onChange={registerField(handleType, {
+          defaultValue:
```

Comment:
This one kind of confirms we should think about the approach proposed in the previous comment 🤓